data/korath/korath missions.txt

```diff
@@ -257,3 +257,598 @@ mission "Small Scale Delivery"
+				launch
+
+			label "second attempt"
+			scene scene/volcanic-eruption-at-night
+			action
+				log "Was able to pass through the atmosphere of Rabasot Pa'ah despite the unusual atmospheric event. A volcanic eruption nearly hit your ship, perhaps caused by landing in the wrong area. There is a large island - or small continent - that looks suitable for landing. Your next landing attempt should succeed"
+			`In this trip, your ship's descent stirred up awe-inspiring storm clouds, but traveling through them was easy. Your shield plow technique took you down to the lower troposphere, where the storms cleared enough for you to see. The original island chain was too far away, so you set course for a larger island nearby.`
+			`	Aiming for a hard-looking piece of ground, you touch down. As this planet has twice Earth's gravity, you extend your ship's artificial gravity field to the surface. Donning a space suit for safety's sake, you walk out to see nature taking control once again: the massive volcano in front of you starts to erupt.`
+			`	After a dash to the cockpit, still in your suit, you take off, satisfied that you've reached the surface - albeit in the worst possible spot. A third attempt will certainly succeed, as you saw a small continent with no apparent volcanic activity, and a pleasant-looking bay.`
+				launch
+
+			label "third attempt"
+			scene scene/blue-whale
+			action
+				set "landed on Rabasot Pa'ah"
+				log "Discovered a creature in a bay on the ocean world of Rabasot Pa'ah with the front half of a whale and the back half of a giant squid. They seem out-of-place, since the planet is almost devoid of macroscopic lifeforms. Perhaps someone is shipping them to the planet?"
+			`On your third attempt, you dodge most of the storms by flying into the atmosphere at an odd angle that keeps you in stable air nearly the whole way. After a long flight through safe atmosphere, you finally reach the bay and land on a durable mesa overlooking it. Extending your ship's artificial gravity field to the surface, you don your suit and walk out. It quickly becomes clear the mesa is artificial: it's a constructed landing pad for a ship of vast scale.`
+			`	This is the perfect location for life, so it's no surprise there are small plants and a few fish in the water. The seas are calmer here than the rest of the planet and an unusual formation in the ionosphere protects this spot from the worst rays of the nova. The dominant land-based lifeform seems to be toothed worms about the size of your arm.`
+			`	The strangest feature is what appears to be slow-moving islands. As one comes closer, its form is more clear: an unholy marriage between a blue whale and a giant squid. On the far end of the mesa, you spy a gravity lift system that looks suitable for lifting such a beast out of the water.`
```

Comment:
```suggestion
			`	The strangest feature is what appears to be slow-moving islands. As one comes closer, its form is more clear: an unholy marriage between a blue whale and a giant squid. On the far end of the mesa, you spy a gravity lift system that looks suitable for carrying such a beast out of the water.`
```

Reply:
Carrying implies horizontal movement as well as vertical, so this changes the meaning. However, it *would* need to be able to move horizontally. I think the correct change is to have a gravity *crane* system that can *carry* the beast. I shall make that change.